src/knock.ts

```diff
@@ -127,6 +127,11 @@ class Knock {
     try {
       return await this.client.put(path, {
         params: options.query,
+        headers: {
+          "Content-Type": "application/json",
+          Accept: "application/json",
+          ...options.headers,
+        },
```

Comment:
I think we're missing these across GET, PUT, and DELETE requests just FYI. It seems that Axios set these as default headers when making requests, but our fetch client does not.

Given our API only speaks JSON, I think we can put this in the client constructor on line 56 cc @connorlindsey

Reply:
In that case, yeah, let's add it to the constructor